react/react1/week2/react1-week2/src/components/TodoList.jsx

```diff
@@ -0,0 +1,23 @@
+import TodoListRow from "./TodoListRow";
+const TodoList =({todos, setTodos}) =>{
+   
+    const deleteItem= (id)=>{
```

Comment:
lets keep the same naming logic, if you used addTodoItem, you should use deleteTodoItem and so on.. 
```suggestion
    const deleteTodoItem= (id)=>{
```